tests/scripts/utils.sh

```diff
@@ -113,26 +100,30 @@ function run_suite() {
   
   install_ols "$1" "$3" "$4" "$5" "$6" "$7" "$8" "$9"
 
+  # CI TEST
+  sleep 10
+  ARTIFACT_DIR=$ARTIFACT_DIR SUITE_ID=$1 python tests/scripts/must_gather.py
+  return 1
+
   # wait for the ols api server to come up
   wait_for_ols "$OLS_URL"
   if [ $? -ne 0 ]; then
     echo "Timed out waiting for OLS to become available"
     echo "${JUNIT_TEMPLATE}" | sed "s/SUITE_ID/${1}/g" > $ARTIFACT_DIR/junit_setup_${1}.xml
 
-    must_gather $1
+    ARTIFACT_DIR=$ARTIFACT_DIR SUITE_ID=$1 python tests/scripts/must_gather.py
```

Comment:
what about moving wait_for_ols into a pytest setup step and then we don't need this separate invocation of must_gather because the teardown will cover it?

would just need to ensure the setup step records a junit failure if it times out (i'm not sure if that's possible from pytest)

Reply:
Should be possible via `unittest`. As it's beyond the scope of OLS-506, I've created a [follow-up story](https://issues.redhat.com/browse/OLS-581) for that.